templates/macros/_macros-image.jinja

```diff
@@ -0,0 +1,22 @@
+# Params
+# caption: Caption for image (required)
+# Slots
+# image_content: Slot for image content.
+
+{%- macro image_span(caption) -%}
+  {%- set image_content = caller('image_content') -%}
+  {%- set has_image = image_content|trim|length>0 -%}
+
+  <section class="p-strip">
+    <div class="row">
+      {% if has_image %}
+        <div class="col u-align--center u-vertically-center u-no-padding--bottom p-image-container--cinematic">
```

Comment:
`p-image-container` requires the image inside to have the class `p-image-container__image`, but at the level of abstraction in this macro, the macro has no control over the class on the image that is passed in.

Probably the caller should be responsible for passing in `p-image-container` or something else

[Docs](https://vanillaframework.io/docs/patterns/images#image-container-with-aspect-ratio)

Reply:
I'll add a note in `_macros-image.jinja` to state that the caller must pass in `p-image-container` for now. I think it would be best practice to set the class of the image being passed if possible, but I'm not sure how to enforce that within this macro. If you have any suggestions please let me know as well